### PR TITLE
Move _test_inductor_realize into python

### PR DESCRIPTION
**Pull request by peterbell10**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #90517

Addresses https://github.com/pytorch/pytorch/pull/90014/files#r1043625932

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Review by ngimel — APPROVED** [review]
Pretty cool that everything can be done in python now

File: torch/_inductor/test_operators.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import torch.library
+from torch import Tensor
+from torch.autograd import Function
+
+_test_lib_def = torch.library.Library("_inductor_test", "DEF")
+_test_lib_def.define("realize(Tensor self) -> Tensor")
+
+_test_lib_impl = torch.library.Library("_inductor_test", "IMPL")
+for dispatch_key in ("CPU", "CUDA", "Meta"):
```
Comment (albanD) [inline torch/_inductor/test_operators.py:9]: Given that you are doing an autograd-aware implementation, you should register this to `AutogradCPU`, `AutogradCUDA` and `AutogradMeta`.

Comment (albanD) [reply torch/_inductor/test_operators.py:9]: My bad I misread this. I though the custom Function was registered on these keys.

**Pull request closed without merging**